src/lucky/force_ssl_handler.cr

```diff
@@ -46,7 +47,18 @@ class Lucky::ForceSSLHandler
 
   private def redirect_to_secure_version(context : HTTP::Server::Context)
     context.response.status_code = settings.redirect_status
+    add_hsts_if_enabled(context)
     context.response.headers["Location"] =
       "https://#{context.request.host}#{context.request.resource}"
   end
+
+  private def add_hsts_if_enabled(context : HTTP::Server::Context)
+    if !!settings.hsts
```

Comment:
Instead of using an if and `not_nil!` what do you think about using `try`?

```crystal
settings.hsts.try do |hsts|
   # Then you can use it without `not_nil!`
end
```